src/server/http/__tests__/xsrf.js

```diff
@@ -104,14 +144,58 @@ describe('xsrf request filter', function () {
         expect(resp.payload).to.be('ok');
       });
 
-      it('rejects requests without either an xsrf or version header', async function () {
+      it('accepts requests with any allowed media type', async function () {
+        const allowedContentTypes = [
```

Comment:
This is great.  I'm really kind of surprised that hapi didn't parse out the charset, to be honest, but that's neither here nor there.